src/components/Guilds/ProposalPage/ProposalVoteCard/ProposalVotes.tsx

```diff
@@ -18,51 +25,42 @@ const VotesRow = styled.div`
   font-size: 14px;
   font-weight: 600;
   margin: 5px 0px 5px 0px;
-  color: ${({ theme, type = 'yes' }) => theme.colors.votes[type].fg};
+  color: ${({ theme, type = '0' }) => theme.colors.votes[type].fg};
 `;
 
 const StyledBullet = styled(Bullet)`
   font-size: 30px;
 `;
 
-export const ProposalVotes = ({ voteData, showToken, token }) => {
-  const { yes, no, totalLocked } = voteData;
-
-  const valueToDisplay = value =>
-    showToken ? value : Math.round((value / totalLocked) * 100);
-  const yesDisplay = valueToDisplay(yes);
-  const noDisplay = valueToDisplay(no);
-
+export const ProposalVotes = ({ showToken, token }: ProposalVotesProps) => {

```

Comment:
I think it would be nice to all use the same pattern for Component props interface. 
```js
export const ProposalVotes: React.FC<ProposalVotesProps> = ({ showToken, token })=> { 
... 
```